### DISCUSSION TITLE
Extracting metadata from .fda files (@PATIENT_INFO_03)

### DISCUSSION THREAD

**Discussion by MathiasGallardo** [marksgraham/OCT-Converter#129] [General]
Hi everyone,

I would like to extract the patient data from some TOPCON DICOMs.
Thanks to the oct-converter library, I could notice that my fda. files include the chunk '@PATIENT_INFO_03', but I did not manage to extract the patient data.
However, I had the feeling that the patient data is included in the FILELIST since we can see these data using the TOPCON viewer.
Did anyone have a look at this problem and have some suggestions?

I saw that this topic has been already brought here during December 2020: https://github.com/marksgraham/OCT-Converter/issues/16
The following reply could be a solution to my problem: https://github.com/marksgraham/OCT-Converter/issues/16#issuecomment-752272973

Many thanks in advance,

Mathias

**Comment by marksgraham** [comment]
Hi,
 
So there is no patient data in the metadata file when you call `.read_all_metadata()`? 

Is this filelist a separate file to the `.fda`, and do you have access to it?

**Reply by MathiasGallardo** [reply]
Hi Mark,

Thanks a lot for your message!
1) When I call .read_all_metadata(), the field "patientext_info" contains {'unknown': [0, 0,...,77]}
Is it the field to look at?
2) Yes, the FILELIST is a separate file from the .fda files. 
I have access to it.

**Reply by marksgraham** [reply]
I have a field `patient_info_02` in my metadata - maybe your fda doesn't contain that?

Afraid I have no experience with the filelist file

**Reply by MathiasGallardo** [reply]
I confirm that I do not have the field patient_info_02.
Thanks for your feedback!
Hope that someone will see my message!

**Comment by witedev** [comment]
Is there any news about @PATIENT_INFO_03 chunk structure?

Thanks in advance
